### PR TITLE
Comments list by article

### PR DESCRIPTION
**Pull request by alexizumi**

### CODE REVIEW

File: src/components/CommentsList.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { CardTitle } from 'reactstrap';
+import { getCommentsByArticle } from '../api/api';
+import CommentCard from './CommentCard';
+import Loading from './Loading';
+
+export default function CommentsList() {
+    const { article_id } = useParams();
+    console.log(article_id, 'article id in CommentsList')
```
Comment (nc-mohammed-dastgir) [inline src/components/CommentsList.jsx:10]: Dont forget to remove console.logs

**Review by nc-mohammed-dastgir — COMMENTED** [review]
Nice logic, but it seems like your api is throwing an error when it should not. When going to an article with no comments (article id 37) we should receive an empty array, but currently your api throws a 404 error message for this

File: src/components/CommentsList.jsx (hunk 1 above)
Comment (alexizumi) [reply src/components/CommentsList.jsx:10]: Done

**Comment by alexizumi** [comment]
> Nice logic, but it seems like your api is throwing an error when it should not. When going to an article with no comments (article id 37) we should receive an empty array, but currently your api throws a 404 error message for this

Will need to come back later on this issue. For now it's working with a workaround

**Pull request merged**